Sources/SwiftFusion/Geometry/Cal3_S2.swift

```diff
@@ -0,0 +1,51 @@
+
+/// The five-parameter camera calibration.
+public struct Cal3_S2: CameraCalibration, Equatable {
+  /// Focal length in X direction.
```

Comment:
not: make this more compact by doing one-line comments after definition, and getting rid of newlines

Reply:
Will the doc generator be able to parse inline documentation comments?
```swift
  public var fx: Double  /// Focal length in X direction.
  public var fy: Double  /// Focal length in Y direction.
  public var s: Double  /// Skew factor.
  public var u0: Double  /// Image center in X direction.
  public var v0: Double  /// Image center in Y direction.
```

Reply:
IDK. Note I can't engage in long review conversations/back and forth. I don't have the time. Just re-request a review when you want me to look at it again, I will do one review now.